docs/scale_run_script.md

```diff
@@ -0,0 +1,71 @@
+# Using the scale run script
+
+The [utils/scale_run.sh](../utils/scale_run.sh) script is used to automate the
+execution of scale testing for the various EC2 instance types.  This script
+runs 5 rounds each of cold start and warm start scenarios.  It creates a
+directory for each run.
+
+Out of the box, it will run the following:
+
+* 1 run of 100 instances on a c5.large instance for the trial license
```

Comment:
Aren't we using m5.large for the trial instance type?

Reply:
:man_facepalming: Yes. Updated